### PR TITLE
Translated the Claim page

### LINKED ISSUE #75: Localize Claim page

### PR DESCRIPTION
**Pull request by alecrem**
Closes #75

### CODE REVIEW

File: pages/claim/index.tsx [hunk 1]
```diff
@@ -47,20 +49,18 @@ const Claim: NextPage = () => {
...
               </Text>
               {hasNFT && claimableToken != undefined ? (
                 <Text>
                   {claimableToken == BigInt(0)
-                    ? 'You currently have no points to claim'
-                    : `You can get ${ethers.utils.formatEther(
-                        claimableToken
-                      )} $HENKAKU`}
+                    ? t('CLAIM_CLAIMABLE_AMOUNT_0')
+                    : t('CLAIM_CLAIMABLE_AMOUNT_1') + ethers.utils.formatEther(claimableToken) + t('CLAIM_CLAIMABLE_AMOUNT_2')}
```
Comment (alecrem) [inline pages/claim/index.tsx:63]: I have no points myself, so I wasn't able to see how this displays on my browser.

**Review by hourglasshoro — APPROVED** [review]
Thank you for the implementation. As I commented a little, LGTM! I tried a little to see if I could translate the error messages displayed in the toast, but it didn't work, so I'll leave it as an issue!

File: public/locales/ja/common.json [hunk 2]
```diff
@@ -46,12 +47,23 @@
   "QUESTS_INPUT_PLACEHOLDER": "キーワード",
   "QUESTS_SUBMIT_BUTTON": "キーワードを送信",
 
+  "CLAIM_HEADING": "$HENKAKUを受けとる",
+  "CLAIM_EXPLANATION_HEADING": "トークンを受け取りましょう。",
+  "CLAIM_EXPLANATION_BODY": "$HENKAKU受けとり可能です。",
+  "CLAIM_EXPLANATION_HEADING_NO_NFT": "まずはNFTを取得しましょう！",
+  "CLAIM_EXPLANATION_BODY_NO_NFT": "$HENKAKUを受け取るには家紋NFTが必要です。",
+  "CLAIM_CLAIMABLE_AMOUNT_0": "しかし、$HENKAKUを受け取れるほどのポイントをお持ちではありません。",
+  "CLAIM_CLAIMABLE_AMOUNT_1": "ただいま",
```
[the remark this answers is not in the document]
Comment (alecrem) [reply public/locales/ja/common.json:56]: Thank you! I don't get much feedback about the translation itself 🥲

I will change it later, but if anyone wants to merge this PR right away or push any text changes, they can feel free to do so.

[the remark this answers is not in the document]
Comment (alecrem) [reply public/locales/ja/common.json]: Done!

File: public/locales/ja/common.json [hunk 3]
```diff
@@ -46,12 +47,23 @@
...
 
+  "CLAIM_HEADING": "$HENKAKUを受けとる",
+  "CLAIM_EXPLANATION_HEADING": "トークンを受け取りましょう。",
+  "CLAIM_EXPLANATION_BODY": "$HENKAKU受けとり可能です。",
+  "CLAIM_EXPLANATION_HEADING_NO_NFT": "まずはNFTを取得しましょう！",
+  "CLAIM_EXPLANATION_BODY_NO_NFT": "$HENKAKUを受け取るには家紋NFTが必要です。",
+  "CLAIM_CLAIMABLE_AMOUNT_0": "しかし、$HENKAKUを受け取れるほどのポイントをお持ちではありません。",
+  "CLAIM_CLAIMABLE_AMOUNT_1": "現在",
+  "CLAIM_CLAIMABLE_AMOUNT_2": "$HENKAKUを受けとることができます。",
+  "CLAIM_SUBMIT_BUTTON": "受けとる",
```
Comment (SamuraiT) [inline public/locales/ja/common.json:58]: can you change to like this structure ?

```json
{
"clam": {
  "header": "hoge",
  "body": "hoge"
}, 
"common": {
 "header": "foobar",
 ....
}
}
```

**Review by SamuraiT — CHANGES_REQUESTED** [review]

File: public/locales/ja/common.json (hunk 3 above)
Comment (alecrem) [reply public/locales/ja/common.json:58]: That's a great idea! But I think we should apply this to the whole string files at once, so I made it a different issue: #84 

This PR is focussed on the Claim page.

Comment (SamuraiT) [reply public/locales/ja/common.json:58]: ok

**Review by SamuraiT — APPROVED** [review]
if everything works fine with testnet, you can merge it

**Comment by alecrem** [comment]
> if everything works fine with testnet, you can merge it

As I said, I haven't been able to see all the possible text outcomes on my browser, but I did rebase this. I think it should be safe to merge, but if there's an easy way to claim a number of times on goerly that would be great to know! Not just for this PR.

**Comment by alecrem** [comment]
Reimplemented in #114

**Pull request closed without merging**